inappmessaging/src/main/java/com/rakuten/tech/mobile/inappmessaging/runtime/manager/MessageReadinessManager.kt

```diff
@@ -94,7 +95,8 @@ internal interface MessageReadinessManager {
                     message.getCampaignId(),
                     HostAppInfoRepository.instance().getVersion(),
                     BuildConfig.VERSION_NAME,
-                    locale?.toString() ?: "",
+                    locale?.toString()?.replace("_", "-")
```

Comment:
It seems like something is missing here. Where does the locale in `HostAppInfoRepository` come from? Doesn't that locale also need to have the `_` replaced? Also, maybe there's a way to ensure that `HostAppInfoRepository` always returns a locale so you don't have to add this string replacement in multiple places.

Reply:
Moved locale string update to `HostAppInfoRepository`. Please check.